src/utils/mutate.js

```diff
@@ -0,0 +1,242 @@
+import { chunk, cloneDeep, flatten, isFunction, mapValues } from 'lodash';
+import debug from 'debug';
```

Comment:
The addition of debug should probably happen across more than just this util if we were going to make use of it

Can we pull that out of this change and we can capture it as a separate PR?

Reply:
done.